docs/topics/modules/planning_development_work.adoc

```diff
@@ -1,65 +1,31 @@
 [id="planning_development_work"]
 = Planning your development work
 
-After creating a space, use the {osio} planner to plan your development work and to ensure efficient execution of your project.
+When you create a space, the *Create an Application* wizard is displayed. Close the wizard, so that you can plan an end to end development workflow for launching and deploying a sample application before you launch the application.
```

Comment:
The latter sentence is getting a bit clunky and therefore not as clear. Maybe something like this would be better: Close the wizard so that you can plan your development workflow before you launch the application wizard.